test/fixtures/input.css

```diff
@@ -1,15 +1,8 @@
-@use postcss-mixins;
-@use postcss-cssnext;
```

Comment:
Was it intentional to remove mixin and other tests?

Reply:
Yes, it was intentional.

> There were a lot of "examples" of plugins used with @use in the css test file, but they would get out of date/deprecated/etc.
> It is easier to only confirm that the @use functionality works with just one plugin, since it is impossible to test that any possible postcss plugin would work there.

My reasoning was that since you could `@use` _any_ postcss plugin, it wasn't necessary to test that `@use` works with each of them. It seems like the goal here should be to confirm that the `@use` plugin is properly enabled, not to test the specific functionality of `@use`.

Definitely open to other opinions here though